account_invoice_product_supplier_price_update/__openerp__.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# © 2016 Chafique DELLI @ Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Account Invoice Product Supplier Price Update',
+    'summary': 'From the supplier invoice, to automatic update all products '
```

Comment:
In the supplier invoice, automatically update all products 
whose unit price on the line is different from the supplier price